ipywidgets/static/widgets/js/widget_bool.js

```diff
@@ -16,7 +16,7 @@ define([
              * Called when view is rendered.
              */
             this.$el
-                .addClass('widget-hbox widget-checkbox');
+                .addClass('ipy-widget widget-hbox widget-checkbox');
```

Comment:
I'd think `ipywidget` would make more sense, since that's the name of the repo, no?